commit 0c6a5dbf6c66480c2e2473c23db9a2ccb7689077
Author: Adam Borowski <kilobyte@angband.pl>
Date:   Wed Jan 22 12:34:07 2020 +0100

    Add missing includes.
    
    According to the C++ standard, std::runtime_error is defined in <stdexcept>.
    That file used to be pulled by another header but that's no more.

diff --git include/memkind_allocator.h include/memkind_allocator.h
index a7036d51e6f9..45bde75768ef 100644
--- include/memkind_allocator.h
+++ include/memkind_allocator.h
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2019 Intel Corporation.
+ * Copyright (C) 2019 - 2020 Intel Corporation.
  * All rights reserved.
  *
  * Redistribution and use in source and binary forms, with or without
@@ -31,6 +31,7 @@
 #include <exception>
 #include <type_traits>
 #include <cstddef>
+#include <stdexcept>
 
 #include "memkind.h"
 
diff --git include/pmem_allocator.h include/pmem_allocator.h
index 964083cc055f..d373e3cf3278 100644
--- include/pmem_allocator.h
+++ include/pmem_allocator.h
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2018 - 2019 Intel Corporation.
+ * Copyright (C) 2018 - 2020 Intel Corporation.
  * All rights reserved.
  *
  * Redistribution and use in source and binary forms, with or without
@@ -30,6 +30,7 @@
 #include <type_traits>
 #include <atomic>
 #include <cstddef>
+#include <stdexcept>
 
 #include "memkind.h"
 
